src/integration.ts

```diff
@@ -143,6 +143,11 @@ export class ProfilingIntegration implements Integration {
           PROFILE_QUEUE.splice(profileIndex, 1);
           const profile = createProfilingEvent(cpuProfile, profiledTransaction);
 
+          if (client.emit) {
+            // @ts-expect-error preprocessEvent hook doesn't exist?
```

Comment:
Weird, any idea why this does not exist here? Should be on both the client and Badeclient types 🤔

Reply:
I'll check, thought it might be because the changes are not in the latest release yet

Reply:
@mydea seems like it was just ts being cryptic due to the unknown Event.